### ISSUE TITLE
Remove jQuery dependency

### ISSUE THREAD

**Issue by ProLoser** [pineconellc/angular-foundation#236]
I was reading your open pull requests, like #117 and other issues and was like "Why do we need this? There is no jQuery dependency". And then I poked around and realize there **is** in fact one.

Which begs the question: _why?_

`angular-ui-bootstrap` does not have one, and even the `$position` module is an attempt to get rid of one, so what code in here still uses it? I feel like that's a fairly critical concern.

**Comment by circlingthesun** [comment]
Unless something recently changed I'm pretty sure you are mistaken. Where
did you find a jquery dependency?

**Comment by ProLoser** [comment]
In bower

**Comment by circlingthesun** [comment]
Right. Jquery is only used in the unit tests. It is not required to use the
library.

**Comment by ProLoser** [comment]
@circlingthesun so make it a dev dependency instead. A lot of people use automated build tools that concatenate all bower repos together.

**Comment by circlingthesun** [comment]
If you look at the repository used by bower https://github.com/pineconellc/angular-foundation-bower/blob/master/bower.json, you'll see that its not actually a depedency.

**Comment by ProLoser** [comment]
Ah. Well... I guess you guys can treat this issue how you see fit.

**Comment by ProLoser** [comment]
I think you should make all the current dependencies in THIS repo dev-dependencies, and whatever is used in the bower repo should be this repo's non-dev dependencies
